spec/controllers/categories_controller_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CategoriesController, type: :controller do
+  describe 'GET #index' do
+    before do
+      Category.create(name: 'Doces', order: 1)
```

Comment:
please move this to inside the test that requires this data. Not all of them need it, and you'll end up creating unnecessary data.